### ISSUE TITLE
Load parquet files

### ISSUE THREAD

**Issue by Gabriel-p** [glue-viz/glue#2416]
**Is your feature request related to a problem? Please describe it:**
Pandas' `parquet` files are not loaded

**Describe the solution you'd like:**
Load `parquet` files

**Comment by Carifio24** [comment]
I agree that the ability to read Parquet files would be nice. It's probably worth investigating whether something like `pyarrow` has any sort of performance gains over `pandas`, but if you're interested in a very minimal example of a Parquet data loader, you can add the snippet below (which requires `pyarrow`) to your glue [config file](http://docs.glueviz.org/en/stable/customizing_guide/configuration.html), which should allow you to load at least basic Parquet files:

```
from glue.config import data_factory
from glue.core.data_factories.helpers import has_extension
from glue.core.data_factories.pandas import panda_process

from pandas import read_parquet

@data_factory(label="Parquet file", identifier=has_extension("parquet"))
def pandas_read_parquet(path, engine="pyarrow", **kwargs):
    df = read_parquet(path, engine=engine)
    return panda_process(df)
```
